index.html

```diff
@@ -1149,12 +1151,54 @@ <h3>Types</h3>
         </p>
       </section>
 
+
+
+      <section>
+        <h3>Credential Subject</h3>
+
+        <p>
+A <a>verifiable credential</a> contains <a>claims</a> about a <a>subject</a>.
+The <a>subject</a> of the credential MUST be expressed using the
+<code>credentialSubject</code> <a>property</a>.
+        </p>
+
+        <dl>
+          <dt>credentialSubject</dt>
+          <dd>
+The value of the <code>credentialSubject</code> <a>property</a> MUST be
```

Comment:
I do not believe it is possible to test conformance to this statement.

Reply:
Fixed by removing the conformance statement.